web/js/debugmode.js

```diff
@@ -111,6 +112,38 @@ function initDebugMode(getStackAtPoint) {
     });
 }
 
+function createSrcLink(pic) {
+    var li = document.createElement("li");
+
+    var preludeLink = "/doc/Prelude.html#v:" + pic.name;
```

Comment:
Interesting idea, but I actually want to get rid of links to Haddock, because they contain a lot of Haskell-isms that CodeWorld should ignore.